api/src/main/java/au/org/aodn/nrmn/restapi/model/db/SpeciesWithAttributes.java

```diff
@@ -0,0 +1,47 @@
+package au.org.aodn.nrmn.restapi.model.db;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.annotation.Immutable;
+
+import javax.persistence.*;
+
+@Entity
+@Immutable
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+@Table(name = "ui_species_attributes")
+public class SpeciesWithAttributes {
+
```

Comment:
are you using this Entity? it has been removed. validatition is using the interface: UiSpeciesAttributes

Reply:
Yes, it's used by SpeciesWithAttributesRepository. This was added in the earlier PR this one was based of. See https://github.com/aodn/nrmn-application/pull/107.

Reply:
This doesn't look like its being used - can we remove it?  We should consolidate species with attributes methods in the process.